notebooks/official/sdk/SDK_AutoML_Forecasting_Model_Training_Example.ipynb

```diff
@@ -0,0 +1,941 @@
+{
```

Comment:
<h3>Create Output BigQuery Dataset -&gt; Create a BigQuery dataset for the batch prediction output</h3><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>